playbooks/eth1_ip.yml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: all
+  become: True
+  tasks:
+    - nmstate_apply:
```

Comment:
Do you consider it a good idea to sync with other network modules?
e.g. 'nmstate_config' ? Or do you think this would not, because the other networking modules are text-file based?
(I am happy with the current powerful approach, just thinking)

Reply:
My example is not idempotent, but this should not block this: http://dpaste.com/1RJMX7Y

Reply:
> Do you consider it a good idea to sync with other network modules?
> e.g. 'nmstate_config' ? Or do you think this would not, because the other networking modules are text-file based?
> (I am happy with the current powerful approach, just thinking)

Since users will have to consult the Nmstate documentation to use this module my idea is to create modules with the same names as the public Nmstate API to simplify things.